src/modules/root/components/router-link.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {FC, useCallback} from "react";
+import {useDispatch} from "react-redux";
+import {actions} from "redux-router5";
+
+interface RouterLinkProps extends React.ButtonHTMLAttributes<HTMLAnchorElement> {
+    routeName: string;
+    params?: {[key:string]: unknown};
+}
+
+export const RouterLink: FC<RouterLinkProps> = ({routeName, params, ...props}) => {
+    const dispatch = useDispatch();
+    const navigateCallback = useCallback(
+        () => dispatch(actions.navigateTo(routeName, params)),
+    [dispatch]
```

Comment:
V deps ti chybí `routeName` a `params`.

Aby to i vypadalo jako odkaz, tak tam můžeme dát `href`. Ideálně přímo URL, můžeš zkusit https://router5.js.org/api-reference#buildurl a nebo jen `href="#"`. V obou případech je pak nutné potlačit výchozí chování v `onClick` callbacku přes `event.preventDefault()`.